packages/web-pkg/src/components/AppTopBar.vue

```diff
@@ -43,6 +43,9 @@
               />
             </oc-drop>
           </template>
+          <span v-if="hasAutosave" class="oc-flex oc-flex-middle">
+            <oc-icon name="refresh" color="white" v-oc-tooltip="autoSaveTooltipText" />
```

Comment:
maybe we have some magic css variable for the color. if you find something good, if not then leave it with `white`. ;-)